.github/workflows/linux_asan.yml

```diff
@@ -0,0 +1,54 @@
+name: Test Address Sanitizer Build (Linux)
+
+on:
+  pull_request:
+    branches:
+      - main
+      - maintenance/**
+
+default:
+  run:
+    shell: bash
+
+env:
+  PYTHON_VERSION: 3.11
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
```

Comment:
I think your job needs a name here. Something like this: https://github.com/numpy/numpy/blob/2e668061db8967de50413f3e201e97edae4c4a75/.github/workflows/linux_meson.yml#L24

Reply:
Yup this passes the lint locally:

```diff
--- a/.github/workflows/linux_asan.yml
+++ b/.github/workflows/linux_asan.yml
@@ -21,34 +21,35 @@ permissions:
  contents: read # to fetch code (actions/checkout)

jobs:
  {+sanitizer:+}
    if: "github.repository == 'numpy/numpy'"
    runs-on: ubuntu-latest
```

Can iterate locally with i.e., `act` tool:

```
Error: workflow is not valid. 'linux_asan.yml': yaml: unmarshal errors:
  line 24: cannot unmarshal !!str `github....` into model.Job
  line 25: cannot unmarshal !!str `ubuntu-...` into model.Job
  line 27: cannot unmarshal !!seq into model.Job
```